src/app/pages/submission-pages/data-page/data-page.component.ts

```diff
@@ -91,6 +95,23 @@ export class DataPageComponent implements OnInit {
 
   }
 
+  onDeleteFile(event, file) {
+    const fileHref = file._links.file.href;
+
+    this.fileService.deleteFile(fileHref).subscribe(
+      (response) => {
+          if (response.status === HttpStatus.NO_CONTENT ) {
+              console.log(`File: ${file.filename} has been succcesfully deleted from the storage.`);
```

Comment:
Should we keep the console log messages or is it for debugging only?

Reply:
I set it to debug level